windows/flutter/generated_plugin_registrant.cc

```diff
@@ -1,20 +0,0 @@
-//
-//  Generated file. Do not edit.
-//
-
-// clang-format off
-
-#include "generated_plugin_registrant.h"
-
-#include <connectivity_plus/connectivity_plus_windows_plugin.h>
-#include <flutter_secure_storage_windows/flutter_secure_storage_windows_plugin.h>
-#include <local_auth_windows/local_auth_plugin.h>
-
-void RegisterPlugins(flutter::PluginRegistry* registry) {
-  ConnectivityPlusWindowsPluginRegisterWithRegistrar(
-      registry->GetRegistrarForPlugin("ConnectivityPlusWindowsPlugin"));
-  FlutterSecureStorageWindowsPluginRegisterWithRegistrar(
-      registry->GetRegistrarForPlugin("FlutterSecureStorageWindowsPlugin"));
-  LocalAuthPluginRegisterWithRegistrar(
-      registry->GetRegistrarForPlugin("LocalAuthPlugin"));
```

Comment:
Hello @fonkamloic 

Sorry about this, I know about not using `git add .` but kinda forgot before I notice I did use it

So thought since it was generate I delete thinking it was the right thing.

Let me fix it, my bad...🤦🏾‍♂️